utils/config/permision.config.ts

```diff
@@ -0,0 +1,15 @@
+import { Prefix } from './indexer.config'
```

Comment:
What this file will be used for?

Reply:
`(permision or indexer).config ?`

Reply:
But `permision.config` -> should mark some features that are not available on some chains -> e.g create on moonsama

`indexer.config` -> {
  one configuration to correctly handle indexers
 }